query_optimizer/tests/execution_generator/CMakeLists.txt

```diff
@@ -17,39 +17,39 @@ add_test(quickstep_queryoptimizer_tests_executiongenerator_create
          "../quickstep_queryoptimizer_tests_ExecutionGeneratorTest"
          "${CMAKE_CURRENT_SOURCE_DIR}/Create.test"
          "${CMAKE_CURRENT_BINARY_DIR}/Create.test"
-         "${CMAKE_CURRENT_BINARY_DIR}/")
+         "${CMAKE_CURRENT_BINARY_DIR}/Create/")
```

Comment:
@zuyu `ctest -j<num-of-threads>`, e.g. running with 80 threads: `ctest -j80`.
I just sent a separate PR #86 for this.